org_status/status_providers/travis.py

```diff
@@ -16,3 +19,10 @@ def get_status(self, repo, host, branch='master'):
             return Status.UNDETERMINED
 
         return self.status_from_badge_svg(badge_result.text)
+
+    @classmethod
+    def get_status_provider_status(cls):
+        response = requests.get(cls.TRAVIS_STATUS_URL)
```

Comment:
Please wrap this code in a try except block and return `None` when an exception is raised. So, we have three states.